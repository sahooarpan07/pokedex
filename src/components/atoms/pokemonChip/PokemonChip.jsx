import {View, Text} from 'react-native';
import React from 'react';
import {styles} from './PokemonChip.styles';
const PokemonChip = ({backgroundColor, value}) => {
  return (
    <View style={[styles.container, {backgroundColor}]}>
      <Text style={styles.value}>{value}</Text>
    </View>
  );
};

export default PokemonChip;
