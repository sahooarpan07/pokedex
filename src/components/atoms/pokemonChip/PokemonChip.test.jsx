/**
 * @format
 */

import 'react-native';
import React from 'react';
import PokemonChip from './PokemonChip';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('should render PokemonChip', () => {
  const tree = renderer.create(<PokemonChip value={'electric'} />).toJSON();

  expect(tree).toMatchSnapshot();
});
