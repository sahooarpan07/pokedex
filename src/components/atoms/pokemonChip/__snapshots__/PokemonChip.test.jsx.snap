// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PokemonChip 1`] = `
<View
  style={
    [
      {
        "borderColor": "#180345",
        "borderRadius": 6,
        "borderWidth": 2,
        "padding": 4,
      },
      {
        "backgroundColor": undefined,
      },
    ]
  }
>
  <Text
    style={
      {
        "color": "#180345",
        "fontSize": 12,
        "fontWeight": "500",
      }
    }
  >
    electric
  </Text>
</View>
`;
